dotnet-core-music-linux/azuredeploy.json

```diff
@@ -0,0 +1,435 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmName": {
```

Comment:
Thanks @bmoore-msft , appreciate the response. I totally get that the solution here is not dependent on the VM name, however this template will be used in a training exercise / as a training mechanism. I think that ‘connection’ made when having control over this value will be worthwhile as participants make the mental transition (datacenter – Azure). Can I get a pass on this one?

The rest of this is good stuff, I’ve learned some new things – Thanks.